metaci/release/migrations/0015_auto_20210407_0218.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.1.7 on 2021-04-07 02:18
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("release", "0014_auto_20210406_0241"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="release",
+            name="change_case_template",
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.PROTECT,
+                to="release.changecasetemplate",
+            ),
+        ),
```

Comment:
This doesn't look right -- it doesn't set null=False and blank=False, so the migrations end up with a final state that doesn't match the model in models.py.

Btw, I merged a PR from Bryan that added a different migration, so you will probably need to add a new merge migration so that the migration system can order the migrations correctly. (see "migration conflicts" in https://www.algotech.solutions/blog/python/django-migrations-and-how-to-manage-conflicts/)